ruby/introduction/installing_ruby.md

```diff
@@ -188,7 +194,7 @@ Now you're ready to install Ruby. We're going to use a tool called `rbenv`, whic
 To install `rbenv`, run the following in your terminal:
 
 ~~~bash
-brew install rbenv
+brew install rbenv ruby-build
```

Comment:
Good idea to add this, as this is what `rbenv` recommend in their official docs. Have never installed it on a Mac so can't really speak to why it's not in there, but parts of this lesson are quite old.